templates/controls/import_project_modal.html

```diff
@@ -0,0 +1,53 @@
+{% load static %}
+<div id="import_project_modal" class="modal" tabindex="-1" role="dialog" aria-labelledby="import_project_modal_title" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                <h2 class="modal-title" id="import_project_modal_title">Select a Project</h2>
+            </div>
+            <form id="project_import_form" action="{% url 'project_import' project.id %}" onsubmit="show_spinner();" role="form" method="POST">
+                {% csrf_token %}
+                <div class="modal-body">
+                    {{ import_project_form.file }}
+                    <br>
+                    <label for="{{ import_project_form.json_content.id_auto }}">{{ import_project_form.json_content.label }}:</label>
+                    <br>
+                    {{ import_project_form.json_content }}
+
+                    <div class="modal-footer">
+                        <input id="import_component_submit" class="btn btn-success btn-submit" type="submit" value="Import"/>
+                        <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+                        {{ import_project_form.importcheck }}
+                        {{ import_project_form.importcheck.label }}
+                        <img id="import_loading_spinner" src="{% static "img/icons/spinner.gif" %}"/>
```

Comment:
Nice! Might as well reuse what we have. This makes me think we should have a base "importer_form" template, and extend it based on which form we want to use (i.e. Component importer or Project importer), but that's probably out of scope for now. It's a refactoring TODO to keep in mind.